qt/scientific_interfaces/EnggDiffraction/EnggDiffFittingPresenter.cpp

```diff
@@ -501,8 +501,8 @@ void EnggDiffFittingPresenter::doFitting(const std::vector<RunLabel> &runLabels,
     } catch (const std::runtime_error &exc) {
       g_log.error() << "Could not run the algorithm EnggFitPeaks successfully."
                     << exc.what();
-      m_view->userError("Could not run the algorithm EnggFitPeaks successfully",
-                        exc.what());
+      // A userError should be used for this message once the threading has been
```

Comment:
Is there an issue to track this?
Given how little the engineering gui is updated there is a danger that this comment will remain far longer than any of us

Reply:
There is now https://github.com/mantidproject/mantid/issues/25186